### ISSUE TITLE
mergeScripts true is safe?? Sorry, I do not think so.

### ISSUE THREAD

**Issue by lovetingyuan** [posthtml/htmlnano#80]
As everyone knows, the order of scripts is important, so I do not think setting `mergeScripts` true is "safe".
eg:
```html
<html>
<head>
<script>window.foo = 'foo'</script>
</head>
<body></body>
<script src="./script-need-foo-variable.js"></script>
<script>window.bar = 'bar'</script>
</html>
```
after minify, `script-need-foo-variable.js` will be broken.
see https://github.com/posthtml/htmlnano/blob/6bb711a0eaf3cbd3f06e0eadfa45272f1cb4139f/lib/presets/safe.es6#L12

**Comment by maltsev** [comment]
Thanks! I fixed it.

**Comment by maltsev** [comment]
I released this fix in the new version [0.2.5](https://github.com/posthtml/htmlnano/releases/tag/0.2.5).
